test/gateway/delayed-base.test.ts

```diff
@@ -0,0 +1,11 @@
+import { OPRollup } from '../../src/op/OPRollup.js';
+import { testOP } from './common.js';
+
+testOP(OPRollup.baseMainnetConfig, {
+  // https://basescan.org/address/0x0C49361E151BC79899A9DD31B8B0CCdE4F6fd2f6
+  slotDataContract: '0x0C49361E151BC79899A9DD31B8B0CCdE4F6fd2f6',
+  // delay by 1 hour
+  // NOTE: to delay longer, Gateway.commitDepth needs to be bigger
+  minAgeSec: 3600,
```

Comment:
For Base we are requesting historical even though finalised data does exist I.E saying get/proof state from last week whereas with OPFault we say '6 hours ago is OK', there isnt finalised data?

Reply:
```
       T=0         1hr     ...     6hr       ...      3.5day
OP      |       Finalized  ==>   Delayed
OPFault |                      Unfinalized   <==    Finalized
```